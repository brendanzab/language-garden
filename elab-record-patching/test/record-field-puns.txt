let _ : fun (A : Type) (B : Type) -> { A : Type; B : Type } :=
  fun A B => { A; B };

let _ : fun (A : Type) (B : Type) -> { A : Type; B : Type } :=
  fun A B => { A := A; B };

let _ : fun (A : Type) (B : Type) -> { A : Type; B : Type } :=
  fun A B => { A; B := B };

Type
