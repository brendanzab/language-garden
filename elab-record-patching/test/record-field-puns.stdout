<input> : Type :=
  let _ :=
    (fun A B := { A := A; B := B }) :
      fun (A : Type) (B : Type) -> { A : Type; B : Type };
  let _ :=
    (fun A B := { A := A; B := B }) :
      fun (A : Type) (B : Type) -> { A : Type; B : Type };
  let _ :=
    (fun A B := { A := A; B := B }) :
      fun (A : Type) (B : Type) -> { A : Type; B : Type };
  Type
